src/main/resources/templates/pdf/legalStatementGrantOfProbate.html

```diff
@@ -256,6 +247,12 @@
     </tr>
     <tr>
         <td colspan="2">
+            <p>{% if case_details.case_data.titleAndClearingType == "TCTPartAllRenouncing" %}
+                The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
+                at the date of death of the deceased have renounced probate.{% elseif case_details.case_data.titleAndClearingType == "TCTPartSuccAllRenouncing" %}
+                The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfSucceededFirm }}
+                that had succeeded to and carried on the practice of the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
+                at the date of death of the deceased have renounced probate.{% endif %}</p>
```

Comment:
i realize this is just being transplanted, but shouldn't the `<p>` and `</p>` be inside the conditional? (otherwise we add a blank paragraph?)

Reply:
I'd also suggest that having this be
```suggestion
            {% if case_details.case_data.titleAndClearingType == "TCTPartAllRenouncing" %}
                <p>The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
                at the date of death of the deceased have renounced probate.</p>
            {% elseif case_details.case_data.titleAndClearingType == "TCTPartSuccAllRenouncing" %}
                <p>The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfSucceededFirm }}
                that had succeeded to and carried on the practice of the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
                at the date of death of the deceased have renounced probate.</p>
            {% endif %}
```
makes it much clearer that there are two options here - i was trying to work out why the conditional on the `None.` text had two values when this seemed only to have one.

Reply:
Yeah I unfortunately squashed the ridiculous number of commits in this PR, which masked my attempt at doing this. The CCD diff report count shows I had a lot of trial and error. 😅 

Essentially, when trying to imitate the existing _way_ of writing conditionals meant document generation was a no go. Correct indentation would go a long way to making it easier to understand why and fix things.